app/src/main/java/com/example/thecocktailsapp/utils/CustomExceptions.kt

```diff
@@ -0,0 +1,4 @@
+package com.example.starwarsmvvm.utils
+
+class NullCocktailResponse(message: String = "People response is null") : Exception(message)
```

Comment:
Change message. Should say "Cocktails" instead of "People"

Reply:
agreed also check the failureResponse to make sure that it also doesn't say people